apps/studio/components/layouts/StorageLayout/StorageMenu.tsx

```diff
@@ -39,12 +52,16 @@ const StorageMenu = () => {
 
   const { data, error, isLoading, isError, isSuccess } = useBucketsQuery({ projectRef: ref })
   const buckets = data ?? []
+  const sortedBuckets =
+    sort === 'alphabetical'
+      ? buckets.sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
related to previous comment, something like:
```suggestion
      ? buckets.sort((a, b) => a.name.toLowerCase().trim().localeCompare(b.name.toLowerCase().trim()))
```
wdyt?

Reply:
okay to do this - but does doing this cover any edge case that the original implementation wouldn't?

Reply:
Should be pretty similar

Reply:
okay gotcha! will just update it :)